examples/pipeline_wav2letter/metrics.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Union
+
+
+def levenshtein_distance(r: Union[str, List[str]], h: Union[str, List[str]]):
```

Comment:
I'd move this into torchaudio before this lands

Reply:
If moving this into the library, docstring needs to be improved with the equation.